coordinator/src/routes.rs

```diff
@@ -121,14 +122,18 @@ pub async fn get_invoice(State(state): State<Arc<AppState>>) -> Result<Json<Stri
     Ok(Json(invoice.to_string()))
 }
 
+// TODO: We might want to have our own ContractInput type here so we can potentially map fields if
```

Comment:
Yes, definitely.

Reply:
This type is actually rather complex, because it holds a `Vec<ContractInputInfo>` which in turn has multiple nested types inside. To create our own model we would have to replicate the complete structure. I wonder if it's worth doing this at the moment. For the MVP it might be good enough to just use the one from `dlc-manager`.